### ISSUE TITLE
Rocky 8.7 and possibly-broken Ansible

### ISSUE THREAD

**Issue by donsizemore** [GlobalDataverseCommunityConsortium/dataverse-ansible#263]
Looks like the AWS Rocky image found the 8.7 update:
`nothing provides (ansible-core >= 2.12.2 with ansible-core < 2.13) needed by ansible-5.4.0-3.el8.noarch`

Apparently `ansible` as an OS package has been deprecated for some time; perhaps it's finally removed. Indeed, I don't see it at https://distro.ibiblio.org/rocky/8.7/AppStream/x86_64/os/Packages/a/

The recommended way to install Ansible is pip, so perhaps it's time to make that jump at https://github.com/GlobalDataverseCommunityConsortium/dataverse-ansible/blob/develop/ec2/ec2-create-instance.sh#L239 or kick the can and switch to `ansible-core`

**Comment by donsizemore** [comment]
The current Vagrantfile seems to be broken, as well:

```
dnf -y --enablerepo=epel install ansible
<snip>
Error: 
 Problem: conflicting requests
  - nothing provides (ansible-core >= 2.12.2 with ansible-core < 2.13) needed by ansible-5.4.0-3.el8.noarch
 ```

**Comment by donsizemore** [comment]
This was fixed with the release of the ansible-6.0 meta-package. Would still be nice to switch to pip, though.
